### ISSUE TITLE
mup deploy error

### ISSUE THREAD

**Issue by SachaG** [arunoda/meteor-up-legacy#12]
```
$ mup deploy

Meteor-UP : Production Quality Meteor Deployments
--------------------------------------------------

Bundling Started: /root/book
Bundling Error:  spawn ENOENT
-------------------STDOUT-------------------

-------------------STDERR-------------------
```

I don't know why my configuration is triggering so many errors! At least I'm stress-testing Meteor Up :)

**Comment by arunoda** [comment]
is this /root/book is the path of the project in the local box?
seems like there is no such path.?

On Mon, Dec 23, 2013 at 12:09 PM, Sacha Greif notifications@github.comwrote:

> $ mup deploy
> 
> ## Meteor-UP : Production Quality Meteor Deployments
> 
> Bundling Started: /root/book
> Bundling Error:  spawn ENOENT
> -------------------STDOUT-------------------
> 
> -------------------STDERR-------------------
> 
> I don't know why my configuration is triggering so many errors! At least
> I'm stress-testing Meteor Up :)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/arunoda/meteor-up/issues/12
> .

## 

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda

**Comment by SachaG** [comment]
Oh you're right, I'm sorry! I didn't read the comments properly and I thought you had to enter the path to the _remote_ directory. I had assumed that mup would simply use the current directory locally. My bad.

**Comment by arunoda** [comment]
That's okay. Seems like I need more docs on that. Will work on that :)

On Mon, Dec 23, 2013 at 10:52 PM, Sacha Greif notifications@github.comwrote:

> Closed #12 https://github.com/arunoda/meteor-up/issues/12.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/arunoda/meteor-up/issues/12
> .

## 

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda
